tests/10-api.spec.js

```diff
@@ -11,4 +12,83 @@ describe('API', () => {
     should.exist(m);
     m.memoize.should.be.a('function');
   });
+  describe('memoize', () => {
+    it('adds an item to the cache', async () => {
+      const m = new lruMemoize.default.LruMemoize();
+      let executedTestFn = false;
+      const testFn = async () => {
+        // simulate an async task
+        await delay(100);
+        executedTestFn = true;
+        return {success: true, timestamp: Date.now()};
```

Comment:
I actually think we should move this out of the test to the top and say call it expectedResult.

```js
const expectedResult = {success: true, timestamp: Date.now()};

///... stuff happens
await delay(100);
executedTestFn = true;
return expectedResult;
```

then after each call we do:

```js
actualResult.should.eql(expectedResult);
```
so we check the object has the same values after being returned.

Reply:
I actually had a revision like the one you suggested, but I opted being explicit on an assertion about the timestamp being the same.  This meant an assertion error if it were to occur would be explicit about which part of the object was not the expected value.  In this case, I prefer the feedback given by asserting on the timestamp value vs the feedback given with a deep equals assertion failed.